### PR TITLE
 Add dependents and suggesters badges

### PR DESCRIPTION
**Pull request by chadicus**

### CODE REVIEW

File: src/Badge/Model/UseCase/CreateDependentsBadge.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PUGX\Badge\Model\UseCase;
+
+use PUGX\Badge\Service\TextNormalizer;
+use PUGX\Badge\Service\NormalizerInterface;
+use PUGX\Badge\Model\PackageRepositoryInterface;
```
Comment (JellyBellyDev) [inline src/Badge/Model/UseCase/CreateDependentsBadge.php]: The namespace is changed in new version. See https://github.com/PUGX/badge-poser/blob/3dcba5bc1d3141d24dec4109369d4e8471b4bdcf/src/Badge/Model/UseCase/CreateVersionBadge.php#L11
Thanks

File: src/Controller/Badge/DependentsController.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class DependentsController.
+ */
+class DependentsController extends Controller
+{
+    /**
+     * License action.
```
Comment (JellyBellyDev) [inline src/Controller/Badge/DependentsController.php]: copy and paste! :P

**Comment by JellyBellyDev** [comment]
Hey @chadicus are you finish? WOW! Thank you!
Can you show me a demo of these badges?
Can you add these badges in configs? [here]( https://github.com/PUGX/badge-poser/blob/release/v3.0.0/config/routes/badge.yaml) and [here](https://github.com/PUGX/badge-poser/blob/d30de0d6196420360342a9c917184196b3dbaa7e/config/services.yaml#L19)
Thanks!

File: src/Controller/Badge/DependentsController.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+     * @param Poser $poser
+     * @param CreateDependentsBadge $createDependentsBadge
+     * @param ImageFactory $imageFactory
+     * @param string $repository repository
+     * @param string $format
+     * @return Response
+     * @Method({"GET"})
+     * @Cache(maxage="3600", smaxage="3600", public=true)
+     */
+    public function dependentsAction(
```
Comment (JellyBellyDev) [inline src/Controller/Badge/DependentsController.php]: Please remove the word `Action`! Thanks!

File: src/Controller/Badge/SuggestersController.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+     * @param Poser $poser
+     * @param CreateSuggestersBadge $createSuggestersBadge
+     * @param ImageFactory $imageFactory
+     * @param string $repository repository
+     * @param string $format
+     * @return Response
+     * @Method({"GET"})
+     * @Cache(maxage="3600", smaxage="3600", public=true)
+     */
+    public function dependentsAction(
```
Comment (JellyBellyDev) [inline src/Controller/Badge/SuggestersController.php]: Please remove the word `Action`! Thanks!

File: src/Controller/Badge/DependentsController.php (hunk 3 above)
Comment (chadicus) [reply src/Controller/Badge/DependentsController.php]: fixed

File: src/Controller/Badge/SuggestersController.php (hunk 4 above)
Comment (chadicus) [reply src/Controller/Badge/SuggestersController.php]: fixed

**Comment by chadicus** [comment]
@JellyBellyDev I'm not sure how to demo the badges. I don't have a PHP 7.1 environment available to me for running the app. The `phpunit/phpunit` is one of the best examples of packages with large numbers of dependents and suggesters.

**Comment by JellyBellyDev** [comment]
@chadicus sorry, but to develop you did not use docker as described in the readme?

**Comment by chadicus** [comment]
@JellyBellyDev  yeah, i had some issues getting the correct docker-compose version installed. I'll keep working on it and try to get a demo to you this weekend

**Comment by JellyBellyDev** [comment]
No problem! I trust!
Thank you for this! LGTM!

**Pull request merged**